sigil.lic

```diff
@@ -0,0 +1,153 @@
+custom_require.call(%w[common common-arcana common-travel drinfomon equipmanager events])
+
+class Sigil
+  include DRC
+  include DRCA
+  include DRCT
+
+  def initialize
+    settings = get_settings
+    @target = variable[1]
```

Comment:
(Not blocking merging) I highly recommend using the `args` framework we provide, instead of `variable`.